sdk/core/core-paging/package.json

```
@@ -35,15 +35,22 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p .",
     "build:test": "tsc -p .",
-    "clean": "",
+    "check-format": "echo skipped",
```

Since "format" exists, "check-format" should also exist.

```suggestion
    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```